docs/guides/1.39.0-upgrade-guide.md

```diff
@@ -0,0 +1,22 @@
+---
+page_title: "Upgrade Guide 1.39.0"
+---
+
+# MongoDB Atlas Provider 1.39.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.39.0 introduces the following new features and fixes:
+
+## New Resources, Data Sources, and Features
+
+- Added support for the `MONGODB_ATLAS_PUBLIC_API_KEY` and `MONGODB_ATLAS_PRIVATE_API_KEY` environment variables which are widely used across the MongoDB ecosystem. These provide an alternative way to configure authentication alongside the existing `MONGODB_ATLAS_PUBLIC_KEY` and `MONGODB_ATLAS_PRIVATE_KEY` variables.
+- The `mongodbatlas_federated_database_instance` resource and related data sources now support Azure cloud provider configuration through the new `azure` attribute. This allows you to create and manage federated databases with Azure. See the [resource documentation](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/federated_database_instance) to learn more.
```

Comment:
Adds support for the `mongodbatlas_federated_database_instance` resource and related data sources in the Azure cloud provider configuration through the new `azure` attribute.

To learn more, see the .... documentation.